WinUIGallery/Controls/PageHeader.xaml

```diff
@@ -48,7 +48,7 @@
                     Visibility="{x:Bind Item.Docs, Converter={StaticResource collectionConverter}}">
                     <DropDownButton.Content>
                         <StackPanel Orientation="Horizontal" Spacing="8">
-                            <FontIcon FontSize="14" Glyph="&#xE8A5;" />
+                            <FontIcon FontSize="16" Glyph="&#xE8A5;" />
                             <TextBlock Text="Documentation" />
```

Comment:
You might want to add a bottom margin (`Margin="0,0,0,2") to the `TextBlock` so it's centered correctly.

Reply:
> looks like it's the link icon

@niels9001 You've got sharp eyes!
I took the image from https://learn.microsoft.com/nl-nl/windows/apps/design/style/segoe-fluent-icons-font#:~:text=Stop-,e71b,-Link
and see that this is by design. If Microsoft designed it like that, I'm nog touching it :)
![image](https://user-images.githubusercontent.com/65828559/231160516-76884ee4-114e-4d06-8fd1-f5e634f9479a.png)

Reply:
It still looks out of place now. So we'd still need to fix the offset.

Reply:
> this is by design. I'm not changing it :)

Reply:
> IIRC Fluent Icons use a 1 pixel thickness for the lines

Try to think of it as **4px** thickness in a **16x16** icon. Then it all adds up. How else would an icon like ![image](https://user-images.githubusercontent.com/65828559/231169179-0a128da3-dc37-4807-9829-4779b468f484.png) be made?

Reply:
Right, what I meant was that these icons were probably designed with a 1px grid. Then you end up with 3 pixels above or below in some edge cases, so it's not perfectly aligned.

<img width="441" alt="image" src="https://user-images.githubusercontent.com/9866362/231172197-03ec12c3-3078-4de0-ac99-b1c4f8bdb011.png">

I think with a simple margin tweak we can visually fix that in context with the other icons?

Reply:
I respectfully refuse. If you insist, you're free to do so in a pr of your own :)